install.sh

```diff
@@ -83,32 +84,34 @@ if [ -e "$BIN_DIR/browserpass" ]; then
   HOST_FILE="$BIN_DIR/browserpass"
 fi
 
-echo ""
-echo "Select your browser:"
-echo "===================="
-echo "1) Chrome"
-echo "2) Chromium"
-echo "3) Firefox"
-echo "4) Vivaldi"
-echo -n "1-4: "
-read BROWSER
-echo ""
+if [ -z "$BROWSER" ]; then
+  echo ""
+  echo "Select your browser:"
+  echo "===================="
+  echo "1) Chrome"
+  echo "2) Chromium"
+  echo "3) Firefox"
+  echo "4) Vivaldi"
+  echo -n "1-4: "
+  read BROWSER
+  echo ""
+fi
 
 # Set target dir from user input
 case $BROWSER in
-1)
+1|[Cc]hrome)
   BROWSER_NAME="Chrome"
   TARGET_DIR="$TARGET_DIR_CHROME"
   ;;
-2)
+2|[Cc]hromium)
   BROWSER_NAME="Chromium"
   TARGET_DIR="$TARGET_DIR_CHROMIUM"
   ;;
-3)
+3|[Ff]irefox)
   BROWSER_NAME="Firefox"
   TARGET_DIR="$TARGET_DIR_FIREFOX"
   ;;
-4)
+4|[Vv]ivaldi)
   BROWSER_NAME="Vivaldi"
   TARGET_DIR="$TARGET_DIR_VIVALDI"
   ;;
```

Comment:
Could you also update the text in `*)` part? Otherwise this would be a confusing behavior:

```
$ ./install.sh chromium-nightly
Invalid selection. Please select 1-4.
```

Reply:
Done. I appended and force pushed to keep the commit history clean.

Reply:
Awesome, thanks 🙂